src/main/resources/application.properties

```diff
@@ -68,7 +68,7 @@ amppd.mediaprobeDir = ${AMPPD_MEDIAPROBE_DIR:../MediaProbe/}
 amppd.encryptionSecret = ${AMPPD_ENCRYPTION_SECRET:secret}
 amppd.admin = ${AMPPD_ADMIN_EMAIL:admin}
 amppd.username = ${AMPPD_USERNAME:username}
-amppd.password = ${AMPPD_PASSWORD:password}
+#amppd.password = ${AMPPD_PASSWORD:password}
 amppd.uiUrl = ${AMPPD_UI_URL:http://localhost:8500/#}
```

Comment:
Also, lets not remove this password property for now. even though we are not using it (i know it was only used by email), we might in the future when we do testing on user session.

Reply:
done